athletics.lic

```diff
@@ -0,0 +1,223 @@
+custom_require.call(%w(common common-travel))
+
+class Athletics
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    @did_play = false
+    @no_instrument = false
+    @settings = get_settings()
+    @athletics_options = get_data('athletics').athletics_options
+    @song_list = get_data('perform').perform_options
+ 
+    Flags.add('ct-song', 'you finish playing')
+    play_song?
+    
+    @start_skill=DRSkill.getxp('Athletics')
+    @end_skill=@start_skill+20
```

Comment:
`@start_skill` is never used again, also why 30 instead of 32?

`@end_skill = [DRSkill.getxp('Athletics')+20, 32].min`